src/Types.jl

```diff
@@ -1354,20 +1360,34 @@ end
 Base.@kwdef struct PackageInfo
     name::String
     version::Union{Nothing,VersionNumber}
+    tree_hash::Union{Nothing,String}
     ispinned::Bool
     isdeveloped::Bool
+    is_tracking_registry::Bool
+    git_revision::Union{Nothing,String}
+    git_source::Union{Nothing,String}
+    git_cache::Union{Nothing,String}
```

Comment:
What is this one storing?

Reply:
the directory in ".julia/clones" where we cache the git repo

Reply:
Do we really want to expose that to users? Isn't that just some function of the `git_source`?

Reply:
Yes, it doesn't seem like something end users should be concerned with. I'll get rid of it